pandas_datareader/yahoo/daily.py

```diff
@@ -97,7 +98,9 @@ def yurl(symbol):
 
     def _get_params(self, symbol):
         unix_start = int(time.mktime(self.start.timetuple()))
-        unix_end = int(time.mktime(self.end.timetuple()))
+        day_end = self.end + datetime.timedelta(days=1) \
```

Comment:
I would add a test like this, to assert the end-points of the received data are correct (should fail before and pass after this PR)

```
import pandas_datareader as pdr
r = pdr.yahoo.daily.YahooDailyReader(['IBM'],'1/1/1960','1/1/2017 12:30:00')
r.end
```